lib/api/api_auth.go

```diff
@@ -97,6 +97,7 @@ func basicAuthAndSessionMiddleware(cookieName string, guiCfg config.GUIConfigura
 
 		if !authOk {
 			emitLoginAttempt(false, username, r.RemoteAddr, evLogger)
+			l.Infoln("Wrong credentials supplied during API authorization")
```

Comment:
You could also include the remote address, then someone wanting to scrape logs can do it directly without the need for `--audit`?

Reply:
I'd also still do this in `emitLoginAttempt`. Reduces ambiguity, more future proof.